14-EventToCommandBehavior/src/PrismSample/Views/EventArgsParameterExamplePage.xaml.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Xamarin.Forms.Xaml;
+using Xamarin.Forms;
+
+namespace PrismSample.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class EventArgsParameterExamplePage : ContentPage
+    {
+        public EventArgsParameterExamplePage()
+        {
+            InitializeComponent();
+        }
+    }
+}
```

Comment:
see comments for EventArgsConverterExamplePage

Reply:
Done with the latest commit. :)